app/jobs/close_patients_job.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# ClosePatientsJob: closes patient records based on criteria
+class ClosePatientsJob < ApplicationJob
```

Comment:
Make sure to update this name here: https://github.com/SaraAlert/SaraAlert/blob/master/README.md#L95, here: https://github.com/SaraAlert/SaraAlert/blob/master/lib/tasks/subjects.rake#L6, and here: https://github.com/SaraAlert/SaraAlert/blob/master/config/schedule.rb#L9

Reply:
Good catch, done.